livetest/citation_test.py

```diff
@@ -38,6 +39,115 @@ def test_printing_as_ris(self):
         self.assertEqual(expected_ris, actual_ris)
 
 
+class CitationTest2(unittest.TestCase):
```

Comment:
Please do not use a number as the test name. When a test fails you now have to look up what the test tests.
Rename from `CitationTest2` to `CitationTestGithubUrlWithSha`.